Files/UserControls/PreviewPane.xaml

```diff
@@ -72,13 +72,13 @@
             <controls:GridSplitter.Element>
                 <Grid>
                     <FontIcon
-                        x:Name="GripperGlyphTextBlock"
+                        x:Name="GripperGlyph"
                         HorizontalAlignment="Center"
-                        VerticalAlignment="Center"
-                        FontSize="36"
                         Foreground="{StaticResource SystemChromeHighColor}"
-                        Glyph="&#xE738;"
-                        IsHitTestVisible="False" />
+                        Glyph="&#xE76F;"
+                        Margin="-1, 0, 0, -1"
```

Comment:
Does it need this margin?

Reply:
@yaichenbaum For some reason the icons don't center without it. Maybe it's a bug in FontIcon?